base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -1226,6 +1226,13 @@
         <type>DEBUG</type>
         <path>twincat/twincat_engineering.opi</path>
         <description>The OPI giving a detailed engineering view of a Beckhoff.</description>
+        <macros>
+        <macro>
+            <name>TC</name>
+            <description>The TC IOC number eg. TC_01</description>
+            <default>TC_01</default>
+          </macro>
+        </macros>
```

Comment:
did you mean to include this?

Reply:
Not really. In fact I am not sure why this block got added when I branched out